.github/workflows/ci.yml

```diff
@@ -61,21 +61,15 @@ jobs:
       run: |
         cabal test all
   stack:
-    name: stack / ghc ${{ matrix.ghc }}
+    name: Stack ${{ matrix.stack }} ${{ matrix.os }}
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        stack: ["2.7.3"]
-        ghc: ["8.10.7"]
```

Comment:
I meant you should only remove this ghc line. the strategy - matrix - stack is still needed I think for the parameters to work properly..

Reply:
Probably it didn't fail earlier because it wasn't being referenced at all. Regardless, I've added it back and removed references of ghc in the stack job.